src/nbl/ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -970,15 +970,11 @@ void CMitsubaLoader::cacheEmissionProfile(SContext& ctx, const CElementEmissionP
 		return;
 	
 	const asset::IAsset::E_TYPE types[]{ asset::IAsset::ET_IMAGE_VIEW, asset::IAsset::ET_TERMINATING_ZERO };
-	auto assetLoaded = ctx.override_->findCachedAsset(profile->filename, types, ctx.inner, 0u);
+	auto assetLoaded = interm_getAssetInHierarchy(m_assetMgr, profile->filename, ctx.inner.params, 0u, ctx.override_);
 	if (assetLoaded.getContents().empty())
```

Comment:
this `interm_` is where you'd override to load metadata only

Reply:
resolved